ion_phys/tests/test_tss.py

```diff
@@ -14,6 +14,27 @@ def _steady_state_population(intensity):
 
 class TestTSS(unittest.TestCase):
     """two-state system tests"""
+    def test_spont_rates(self):
+        """Test the spontaneous rates satisfy relations in net rates"""
+        intensity_list = [1e-3, 1e-1, 0.3, 1, 1.0, 2, 10., 1.2e4]
+
+        ion = Ca43(B=5e-4, level_filter=[ground_level, P32])
+        s_idx = ion.index(ground_level, 4)
+        p_idx = ion.index(P32, +5)
+
+        rates = Rates(ion)
+        delta = ion.delta(s_idx, p_idx)
+        for I in intensity_list:
+            Lasers = [Laser("393", q=+1, I=I, delta=delta)]  # resonant
+            trans = rates.get_transitions(Lasers)
+
+            spont = rates.get_spont()
+            self.assertAlmostEqual(1.,
+                spont[p_idx, p_idx]
+                / (trans[p_idx, p_idx] + trans[p_idx, s_idx]),
```

Comment:
I don't understand the syntax here. Missing `,`?

Reply:
This is correct. The call signature is: `(1., spont[p_idx, p_idx] / (trans[p_idx, p_idx] + trans[p_idx, s_idx]), places=7)`. The linebreak in the second argument is valid as it's within the call signature.

Reply:
Okay, I see, you're comparing the ratio to one rather than comparing them to each other directly to allow you to pick a scale-invariant epsilon. Fine!